tests/test_envs.py

```diff
@@ -32,16 +43,47 @@ class TestEnvs:
     """Battery of simple tests for environments."""
 
     def test_seed(self, env: gym.Env, env_name: str):
-        """Tests environment seeding."""
-        envs.test_seed(env, env_name, DETERMINISTIC_ENVS)
+        """Tests environment seeding.
+
+        Deterministic atari environments are run with fewer seeds to minimize the number
+        of resets done in this test suite, since atari resets take a long time and there
+        are many atari environments.
+        """
+        if env_name in ATARI_ENVS:
+            rollout_len = (
+                100
+                if env_name not in ["seals/Bowling-v5", "seals/NameThisGame-v5"]
+                else 400
+            )
+            # those two environments take a while for their non-determinism to show.
```

Comment:
You're referring to the environments on line 55? Consider defining a variable for those and adding the comment before that variable. I think this would actually make the code *shorter* as you could then compact line 53 `rollout_len = ...` into a single line.

Reply:
Will do, altho I dispute that that's the correct way to measure code length.